lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
Since the schema is the only thing we can _actually_ use for validation, I consider the schema to be the authoritative source. If the schema is wrong, they need to fix it.

I still think empty meta is the wrong way to convey the concept of unknown relationships.